governance/GOVERNANCE.md

```diff
@@ -0,0 +1,116 @@
+# ORAS Governance
+
+The following document outlines ORAS project governance.
+
+## The ORAS Project
+
+The ORAS project consists of several repositories known as sub-projects that enable community cohorts to experiment and implement solutions across the scope of the project.
+
+## Owners Structure
+
+There are two types of owners in the ORAS project organized hierarchically. ORAS [org owners][oras-project-owners] oversee the overall project and its health. Subproject owners focus on a single repository, a group of related repositories, a service (e.g., a website), or subproject to support the other subprojects (e.g., marketing or community management). 
+
+Changes in Oras Org owners have to be announced via an [oras-project/community/issues][oras-issues-new]. Changes to sub-project owners are to be announced via the appropriate sub-project issue.
+
+### Owner Responsibility
+
+ORAS owners adhere to the requirements and responsibilities set forth in the respective [ORAS Org Owners](#oras-org-owners) and [Subproject Owners](#subproject-owners). They further pledge the following:
+
+- To act in the best interest of the project and subprojects
+- To ensure that project and subproject development and direction is a function of community needs
+- To never take any action while hesitant that it is the right action to take
```

Comment:
Can we change this from "never" to "should not"? Rigid language can bind us in unforeseen ways.

Reply:
I struggled for how to reword this, as it sounded like a double negative.
> - To never take any action while hesitant that it is the right action to take

- Should not take any action while hesitant that it is the right action to take

Sounds like they shouldn't take any action?

@lachie83, can you help with some wording? I'll do another PR to amend this if we can find better language.

Reply:
"Should only take action when reasonably confident that it is the right action to take"?

Reply:
I've tweaked it a minor bit, to be consistent with the other wording:
To take action when reasonably confident that it is the right action to take